R/pkg/R/functions.R

```diff
@@ -3083,11 +3011,10 @@ setMethod("rank",
             column(jc)
           })
 
-# Expose rank() in the R base package
-#' @param x a numeric, complex, character or logical vector.
-#' @param ... additional argument(s) passed to the method.
-#' @name rank
-#' @rdname rank
+#' @details
+#' \code{rank}: Exposes \code{rank()} in the R base package. In this case, \code{x}
```

Comment:
Ye, actually we don need to doc this. We only need to add an alias and that should satisfy the R requirement. User can still use the base doc.